# Generated by Django 4.2.3 on 2023-09-30 18:15

import datetime
import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('core', '0017_alter_filmoftheweek_end_date_and_more'),
    ]

    operations = [
        migrations.AddField(
            model_name='filmoftheweek',
            name='summary_picture_path',
            field=models.TextField(blank=True),
        ),
        migrations.AlterField(
            model_name='log',
            name='date',
            field=models.DateField(default=datetime.date.today, validators=[django.core.validators.MaxValueValidator(datetime.date(2023, 9, 30), message='You have entered a date that is beyond the current timeline. Please check your flux capacitor and try again.')]),
        ),
    ]
